publisher_joint_trajectory_controller:
  ros__parameters:

    controller_name: "joint_trajectory_controller"
    wait_sec_between_publish: 5
    
    joints:
      - shoulder_pan_joint
      - shoulder_lift_joint
      - elbow_joint
      - wrist_1_joint
      - wrist_2_joint
      - wrist_3_joint

    goal_names: ["pos1", "pos2", "pos3", "pos4"]
    pos1: [0.785, -1.57, 0.785, 0.785, 0.785, 0.785]
    pos2: [0.0, -1.57, 0.0, 0.0, 0.0, 0.0]
    pos3: [0.0, -1.57, 0.0, 0.0, -0.785, 0.0]
    pos4: [0.0, -1.57, 0.0, 0.0, 0.0, 0.0]